libs/designsystem/src/lib/components/calendar/calendar.component.spec.ts

```diff
@@ -102,6 +102,22 @@ describe('CalendarComponent', () => {
     verifyMonthAndYear('January 1998');
   });
 
+  it('should emit the previousMonthClicked event, when navigating to a past month', () => {
+    const previousMonthClickedSpy = spyOn(spectator.component.previousMonthClicked, 'emit');
+
```

Comment:
I would verify that the event hasn't been called before you call it yourself, as this can happen, leading to a false positive: 
``` Typescript
    expect(previousMonthClickedSpy).not.toHaveBeenCalled();
```

Reply:
Done :)